docs/Project.toml

```diff
@@ -1,2 +1,3 @@
 [deps]
 Documenter = "e30172f5-a6a5-5a46-863b-614d45cd2de4"
+MetaGraphsNext = "fa8bd995-216d-47f1-8a91-f3b68fbeb377"
```

Comment:
```suggestion
```

you should not need it, to build locally the best is to dev the main package before building the docs

Reply:
That's what I did, and it shows up in the Manifest but not in the Project, can't figure out why